test/versioned/otel-bridge/aws-sqs.test.js

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const assert = require('node:assert')
+const test = require('node:test')
+const otel = require('@opentelemetry/api')
+const helper = require('../../lib/agent_helper')
+const common = require('../aws-sdk-v3/common')
+const { createResponseServer, FAKE_CREDENTIALS } = require('../../lib/aws-server-stubs')
+const sinon = require('sinon')
+const { match } = require('../../lib/custom-assertions')
+
+const AWS_REGION = 'us-east-1'
+
+test('SQS API', async (t) => {
+  t.beforeEach(async (ctx) => {
+    ctx.nr = {}
+    const server = createResponseServer()
+
+    await new Promise((resolve) => {
+      server.listen(0, resolve)
+    })
+
+    ctx.nr.server = server
+    ctx.nr.agent = helper.instrumentMockedAgent({
+      feature_flag: {
+        opentelemetry_bridge: true
+      },
+      instrumentation: {
+        // TODO: is this the correct way to disable the instrumentation?
+        '@aws-sdk/client-sqs': {
```

Comment:
no. it goes off the key name in `lib/instrumentations.js` so in this case it's just `aws-sdk`